Casks/applepi-baker.rb

```diff
@@ -7,6 +7,11 @@
   desc "Backup and restore SD cards, USB drives, external HDD, etc"
   homepage "https://www.tweaking4all.com/hardware/raspberry-pi/applepi-baker-v2/"
 
+  livecheck do
+    url :homepage
+    regex(%r{/downloads/raspberrypi/ApplePi-Baker-v(.*?)\.dmg"}i)
```

Comment:
```suggestion
    regex(/href=.*?ApplePi-Baker[._-]v?(\d+(?:\.\d+)+)\.dmg/i)
```

Besides previously mentioned changes:

* Use the standard regex for versions like `1.2`/`1.2.3`/`v1.2`/etc. (`v?(\d+(?:\.\d+)+)`).
* Omits the trailing `"` since it didn't seem technically necessary. However, when it's necessary to target the end of the attribute, we use `["' >]`.